RoddyCore/src/de/dkfz/roddy/core/Analysis.groovy

```diff
@@ -32,142 +32,145 @@ import static de.dkfz.roddy.tools.RoddyIOHelperMethods.getStackTraceAsString
  * TODO Enable a initialize (cache erase?) to make it possible to reset data sets and data set states for an analysis.
  */
 @CompileStatic
-public class Analysis {
+class Analysis {
 
-    private static final de.dkfz.roddy.tools.LoggerWrapper logger = de.dkfz.roddy.tools.LoggerWrapper.getLogger(Analysis.class.getSimpleName());
+    private static final de.dkfz.roddy.tools.LoggerWrapper logger = de.dkfz.roddy.tools.LoggerWrapper.getLogger(Analysis.class.getSimpleName())
 
     /**
      * An analysis should have a unique name like i.e. whole_genome_processing or exome_analysis
      */
-    private final String name;
+    private final String name
 
     /**
      * An analysis has a single link to a project, but the project can hav multiple analysis objects.
      */
-    private final Project project;
+    private final Project project
 
     /**
      * An analysis is directly linked to a configuration
      */
-    private final Configuration configuration;
+    private final Configuration configuration
 
     /**
      * The basic input directory for an analysis. This directory should then contain a list of datasets.
      * The list of datasets is created by the projects runtime service.
      */
-    private File inputBaseDirectory;
+    private File inputBaseDirectory
 
     /**
      * Runtime service is set in the analysis config. But the user can also set it for a project. The project then goes first, afterwards the analysis.
      */
-    private RuntimeService runtimeService;
+    private RuntimeService runtimeService
 
-    public Analysis(String name, Project project, RuntimeService runtimeService, AnalysisConfiguration configuration) {
-        this.name = name;
-        this.project = project;
-        this.configuration = configuration;
-        this.runtimeService = runtimeService;
+    Analysis(String name, Project project, RuntimeService runtimeService, AnalysisConfiguration configuration) {
+        this.name = name
+        this.project = project
+        this.configuration = configuration
+        this.runtimeService = runtimeService
     }
 
-    public String getName() {
-        return name;
+    String getName() {
+        return name
     }
 
-    public Project getProject() {
-        return project;
+    Project getProject() {
+        return project
     }
 
-    public String getUsername() {
+    String getUsername() {
         try {
-            return FileSystemAccessProvider.getInstance().callWhoAmI();
+            return FileSystemAccessProvider.getInstance().callWhoAmI()
         } catch (Exception e) {
-            return "UNKNOWN";
+            return "UNKNOWN"
```

Comment:
BTW: Did this ever throw an exception? And if it does, is this not rather a real exception and should not be ignored like here?

Reply:
?